### DISCUSSION TITLE
motion_gen_reacher.py return with "Opt Fail"

### DISCUSSION THREAD

**Discussion by simanxu** [NVlabs/curobo#55] [Software Q&A]
1. cuRobo installation mode: isaac sim
2. python version: 3.10
3. Isaac Sim version (if using): 2022.2.1

I tried motion_gen_reacher.py `omni_python examples/isaac_sim/motion_gen_reacher.py --robot /path/to/kinova_gen3.yml --visualize_spheres
`

```
            result = motion_gen.plan_single(cu_js.unsqueeze(0), ik_goal, plan_config)
            
            succ = result.success.item()
            print(result.status, result.success.item())
```
and always got "Opt Fail" by print(result.status, result.success.item())

I don't know how to solve this error,
------
Issue Details

Thanks

**Comment by balakumar-s** [comment]
Did you try moving the target cube to different positions?

**Reply by simanxu** [reply]
Yes, I have tried different positions, and always got the same output.
Printed info was generated by the follow code
```
            result = motion_gen.plan_single(cu_js.unsqueeze(0), ik_goal, plan_config)        
    
            succ = result.success.item()
            // print solve info here
            print(result.status, result.success.item(), result.trajopt_time, result.trajopt_attempts)
```

![20231128-201621](https://github.com/NVlabs/curobo/assets/22866495/9bb42f58-c80e-4739-ad09-d593015e37b9)

If I ignored the 'Opt Fail' error, the robot can still move to the target cube positions.

**Reply by simanxu** [reply]
As mentioned above
